openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -374,3 +526,75 @@ def test_get_enrollment_url(self, get_auth_token_mock):
         expected_url = '{}/{}'.format('test-instance-url', settings.SALESFORCE_ENROLLMENT_API_PATH)
         self.assertEqual(expected_url, self.base._get_enrollment_url({}))  # pylint: disable=protected-access
         get_auth_token_mock.assert_called_once()
+
+    def test_get_program_course_runs_with_data(self):
+        """This test validates _get_program_course_runs with data."""
+        data = {
+            'program': {
+                'courses': [
+                    {
+                        'course_runs': [
+                            {'key': 'course-test1', },
+                            {'key': 'course-test2', },
```

Comment:
You don't need trailing commas there, please check that througuot the file.
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas.
```suggestion
                            {'key': 'course-test1'},
                            {'key': 'course-test2'},
```